README.md

```diff
@@ -2,9 +2,72 @@
 
 An operator to deploy the [OpenStack Cinder CSI driver](https://github.com/openshift/cloud-provider-openstack/tree/master/pkg/csi/cinder) in OpenShift.
 
-# Quick start
+## Configuration
 
-Before running the operator manually, you must remove the operator installed by CSO/CVO
+The operator can be configured using a config map, which by default is found at either `openshift-config / cinder-csi-config` or `openshift-config / cloud-provider-config`.
+The former is preferred as it stores configuration solely for the Cinder CSI Driver.
+It is supported starting in OpenShift 4.14 and should be used in all new deployments.
+As the name might suggest, the latter stores configuration for both the Cinder CSI driver and the OpenStack Cloud Provider.
+It is used for legacy reasons.
+
+Configuration must be stored in the config map under the `config` key. For example:
+
+```shell
+oc get configmap -n openshift-config cinder-csi-config -o yaml
+```
+
+```yaml
+apiVersion: v1
+data:
+  ca-bundle.pem: |
+    <redacted>
+  config: |
+    [Global]
+    ...
+    [BlockStorage]
+    ...
+kind: ConfigMap
+metadata:
+  name: cinder-csi-config
+  namepsace: openshift-config
+```
+
+> *Note*
+> This config map stores configuration for both services for historical reasons: previously, block device management was handled by the cloud provider.
+> This was decoupled in the 4.14 release, but support for the legacy config map is retained to avoid breaking exist deployments.
+> Only values from the `[Global]`, `[BlockStorage]` and `[Metadata]` sections are relevant. The remainder are ignored by the CSI driver.
+> A fully list of supported configuration options can be found in the [OpenStack Cloud Provider documentation](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/using-cinder-csi-plugin.md#driver-config).
+
+> *Note*
+> The name of the legacy config map, `cloud-provider-config`, is configurable and is derived from the `infrastructure / cluster` CRD.
```

Comment:
Is `.spec.cloudConfig.name` pointing to a configmap the contents of which may be overwritten by creating a new config map with name `cinder-csi-config`? Or is `.spec.cloudConfig.name` defining where the cinder csi operator will fetch its whole configuration, full stop?

In other words: if I want to use the new shiny `cinder-csi-config` way of doing things, shall I set `.spec.cloudConfig.name: cinder-csi-config` in the `infrastructure / cluster` CRD?

Also: does setting `.spec.cloudConfig.name` in the `infrastructure / cluster` CRD affects how other operators (and in particular, the cloud provider) behave? If so, I would repurpose this section to be more explicitly a warning that sounds like "the CPO cm name is configurable; if you have changed it for CPO, then the csi operator will also source its legacy cm using that name. This does not apply if using the new shiny `cinder-csi-config` cm."